### PR TITLE
Timeout

### PR DESCRIPTION
**Pull request by drowsy810301**
auto log out after one hour

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -160,6 +160,8 @@ def user_login(request):
             user.backend = 'django.contrib.auth.backends.ModelBackend'
             ip = get_client_ip(request)
             logger.info('user %s @ %s logged in' % (str(user), ip))
+            one_hour = 60 * 60
+            request.session.set_expiry(one_hour)
```
Comment (nickelcheng) [inline users/views.py]: I prefer to add a log message, similar to logout.

**Comment by nickelcheng** [comment]
+1 besides the comment.
@bruce3557 please check

**Comment by drowsy810301** [comment]
comment fixed

**Comment by drowsy810301** [comment]
please review again,thanks

**Comment by bruce3557** [comment]
+1, going to merge

**Pull request merged**